cookbook/security/pre_authenticated.rst

```diff
@@ -11,6 +11,14 @@ box, Symfony supports most authentication mechanisms.
 These requests are called *pre authenticated* requests because the user is already
 authenticated when reaching your application.
 
+.. caution::
+
+    :doc:`User impersonation </cookbook/security/impersonating_user>` is not
+    compatible with pre Authenticated firewalls. The reason is that
```

Comment:
Ok fixed. I missed that rule about section titles
However, shouldn't the title be either

> Using pre-Authenticated Security Firewalls

or

> Using Preauthenticated Security Firewalls